tests/units/Telemetry.php

```diff
@@ -180,9 +180,14 @@ public function testGrabPhpInfos() {
    }
 
    public function testGrabOsInfos() {
+      $distro = '';
+      if (file_exists('/etc/redhat-release')) {
+         $distro = preg_replace('/\s+$/S', '', file_get_contents('/etc/redhat-release'));
+      }
+
```

Comment:
Having to copy/paste main code here seems a bit fragile ?
(having to update when each distro will drop its code)

```
$actual = \Telemetry::grabOsInfos()
$this->array($actual)->haveKey('distribution']); // don't know exact contraint name
$actual['distribution'] = '';
$this->array($actual)->isIdenticalTo($expected);
```